python/tests/graph/test_utils.py

```diff
@@ -0,0 +1,191 @@
+    yield TestCase(data=(op_name, tfx.op_name(tnsr_name, graph)),
+                   description='get op name from tensor name (with graph)')
+    yield TestCase(data=(op_name, tfx.op_name(tnsr.op)),
+                   description='get op name from op (no graph)')
+    yield TestCase(data=(op_name, tfx.op_name(tnsr.op, graph)),
+                   description='get op name from op (with graph)')
+    yield TestCase(data=(op_name, tfx.op_name(op_name)),
+                   description='get op name from op name (no graph)')
+    yield TestCase(data=(op_name, tfx.op_name(op_name, graph)),
+                   description='get op name from op name (with graph)')
+
+    # Test for tensor_name
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr)),
+                   description='get tensor name from tensor (no graph)')
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr, graph)),
+                   description='get tensor name from tensor (with graph)')
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr_name)),
+                   description='get tensor name from tensor name (no graph)')
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr_name, graph)),
+                   description='get tensor name from tensor name (with graph)')
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr.op)),
+                   description='get tensor name from op (no graph)')
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr.op, graph)),
+                   description='get tensor name from op (with graph)')
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr_name)),
+                   description='get tensor name from op name (no graph)')
+    yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr_name, graph)),
+                   description='get tensor name from op name (with graph)')
+
+    # Test for get_tensor
+    yield TestCase(data=(tnsr, tfx.get_tensor(tnsr, graph)),
+                   description='get tensor from tensor')
+    yield TestCase(data=(tnsr, tfx.get_tensor(tnsr_name, graph)),
+                   description='get tensor from tensor name')
+    yield TestCase(data=(tnsr, tfx.get_tensor(tnsr.op, graph)),
+                   description='get tensor from op')
+    yield TestCase(data=(tnsr, tfx.get_tensor(op_name, graph)),
+                   description='get tensor from op name')
+
+    # Test for get_op
+    yield TestCase(data=(tnsr.op, tfx.get_op(tnsr, graph)),
+                   description='get op from tensor')
+    yield TestCase(data=(tnsr.op, tfx.get_op(tnsr_name, graph)),
+                   description='get op from tensor name')
+    yield TestCase(data=(tnsr.op, tfx.get_op(tnsr.op, graph)),
+                   description='get op from op')
+    yield TestCase(data=(tnsr.op, tfx.get_op(op_name, graph)),
+                   description='test op from op name')
+
+    # Test get_tensor and get_op returns tensor or op contained in the same graph
```

Comment:
these tests don't seem necessary now that we have the asserts you've most recently added (we invoke the same function calls in other tests and they will fail if the asserts fail). let's remove them to save on testing and reading time. i like this new way of having the asserts in the methods much better than the tests here, since what we're checking here are really implementation details.